py/__init__.py

```diff
@@ -168,7 +188,11 @@ class PathHolders:
     "PagingPredicateHolder": [PathHolders.PagingPredicateHolder, PathHolders.PagingPredicateHolderCodec],
     "EndpointQualifier": [PathHolders.EndpointQualifier, PathHolders.EndpointQualifierCodec],
     "Map_EndpointQualifier_Address": [PathHolders.MapCodec, PathHolders.EndpointQualifierCodec,
-                                      PathHolders.AddressCodec]
+                                      PathHolders.AddressCodec],
+    "SqlQueryId": [PathHolders.SqlQueryId, PathHolders.SqlQueryIdCodec],
+    "SqlColumnMetadata": [PathHolders.SqlColumnMetadata, PathHolders.SqlColumnMetadataCodec],
+    "SqlError": [PathHolders.SqlError, PathHolders.SqlErrorCodec],
+    "SqlPage": [PathHolders.SqlPageCodec],
```

Comment:
Don't we need `PathHolders.SqlPage` here after adding a SqlPage PathHolders?

Reply:
No, as this is a built-in codec